crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -149,7 +149,11 @@ def ceph_status
         return ret if ceph_nodes.empty?
         ceph_node = ceph_nodes.first
 
+        # FIXME: do I need to pick ceph-mon node? (It might happen that this is just ceph-osd, without the mon role)
```

Comment:
@tserong What about this one? Do I need to pick ceph-mon node to run that health check command?